Index: .idea/hamkor_auto_tests.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<module type=\"PYTHON_MODULE\" version=\"4\">\n  <component name=\"NewModuleRootManager\">\n    <content url=\"file://$MODULE_DIR$\">\n      <excludeFolder url=\"file://$MODULE_DIR$/venv\" />\n    </content>\n    <orderEntry type=\"inheritedJdk\" />\n    <orderEntry type=\"sourceFolder\" forTests=\"false\" />\n  </component>\n</module>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/hamkor_auto_tests.iml b/.idea/hamkor_auto_tests.iml
--- a/.idea/hamkor_auto_tests.iml	(revision 6573788c32752b565da166951987a1d80e503371)
+++ b/.idea/hamkor_auto_tests.iml	(date 1663234723944)
@@ -2,6 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$" isTestSource="false" />
       <excludeFolder url="file://$MODULE_DIR$/venv" />
     </content>
     <orderEntry type="inheritedJdk" />
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"ProjectRootManager\" version=\"2\" project-jdk-name=\"Python 3.10 (hamkor_auto_tests)\" project-jdk-type=\"Python SDK\" />\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 6573788c32752b565da166951987a1d80e503371)
+++ b/.idea/misc.xml	(date 1663223497467)
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.10 (hamkor_auto_tests)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (hamkorautotests) (2)" project-jdk-type="Python SDK" />
 </project>
\ No newline at end of file
Index: card_service/tests/test_cases/ivan_test_case.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from unittest import TestCase\n\nimport allure\n\nfrom api_mobile.test_data.client import Client, User, Device\nfrom card_service.tests.scenarios.info_scenarios import card_info_scenario\n\n\nclass DemoScenarioTestCase(TestCase):\n    def setUp(self) -> None:\n        self.client = Client(\n            User(\n                \"998941775859\",\n                \"8600120480409831\",\n                \"0923\",\n                residence_of_uz=False\n            ),\n            Device(\n                phone_type=\"1\",\n                device_id=\"string7\",\n                device_info=\"string7\",\n                device_os=\"Android\",\n                lang_id=\"ru\"\n            )\n        )\n        self.client.confirm_method = \"SMS\"\n\n    def test_main_page(self):\n        card_info_scenario(self.client)\n\n        with allure.step(f\"{self.client.cards}\"):\n            pass
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/card_service/tests/test_cases/ivan_test_case.py b/card_service/tests/test_cases/ivan_test_case.py
--- a/card_service/tests/test_cases/ivan_test_case.py	(revision 6573788c32752b565da166951987a1d80e503371)
+++ b/card_service/tests/test_cases/ivan_test_case.py	(date 1663319592479)
@@ -2,7 +2,7 @@
 
 import allure
 
-from api_mobile.test_data.client import Client, User, Device
+from back_mobile.test_data.client import Client, User, Device
 from card_service.tests.scenarios.info_scenarios import card_info_scenario
 
 
diff --git a/api_mobile/response_data_types/auth/__init__.py b/back_mobile/response_data_types/auth/__init__.py
rename from api_mobile/response_data_types/auth/__init__.py
rename to back_mobile/response_data_types/auth/__init__.py
diff --git a/api_mobile/tests/scenarios/auth_scenarios.py b/back_mobile/tests/scenarios/auth_scenarios.py
rename from api_mobile/tests/scenarios/auth_scenarios.py
rename to back_mobile/tests/scenarios/auth_scenarios.py
diff --git a/api_mobile/response_data_types/payment/p2p_validate_result.py b/back_mobile/response_data_types/payment/p2p_validate_result.py
rename from api_mobile/response_data_types/payment/p2p_validate_result.py
rename to back_mobile/response_data_types/payment/p2p_validate_result.py
diff --git a/api_mobile/response_data_types/payment/p2p_confirm_result.py b/back_mobile/response_data_types/payment/p2p_confirm_result.py
rename from api_mobile/response_data_types/payment/p2p_confirm_result.py
rename to back_mobile/response_data_types/payment/p2p_confirm_result.py
diff --git a/api_mobile/response_data_types/payment/__init__.py b/back_mobile/response_data_types/payment/__init__.py
rename from api_mobile/response_data_types/payment/__init__.py
rename to back_mobile/response_data_types/payment/__init__.py
diff --git a/api_mobile/response_data_types/payment/templates.py b/back_mobile/response_data_types/payment/templates.py
rename from api_mobile/response_data_types/payment/templates.py
rename to back_mobile/response_data_types/payment/templates.py
diff --git a/api_mobile/response_data_types/settings/language.py b/back_mobile/response_data_types/settings/language.py
rename from api_mobile/response_data_types/settings/language.py
rename to back_mobile/response_data_types/settings/language.py
diff --git a/api_mobile/tests/scenarios/__init__.py b/back_mobile/tests/scenarios/__init__.py
rename from api_mobile/tests/scenarios/__init__.py
rename to back_mobile/tests/scenarios/__init__.py
diff --git a/api_mobile/response_data_types/settings/__init__.py b/back_mobile/response_data_types/settings/__init__.py
rename from api_mobile/response_data_types/settings/__init__.py
rename to back_mobile/response_data_types/settings/__init__.py
diff --git a/api_mobile/tests/test_cases/__init__.py b/back_mobile/tests/test_cases/__init__.py
rename from api_mobile/tests/test_cases/__init__.py
rename to back_mobile/tests/test_cases/__init__.py
diff --git a/api_mobile/response_data_types/main_page/__init__.py b/back_mobile/response_data_types/main_page/__init__.py
rename from api_mobile/response_data_types/main_page/__init__.py
rename to back_mobile/response_data_types/main_page/__init__.py
diff --git a/api_mobile/response_data_types/references/__init__.py b/back_mobile/response_data_types/references/__init__.py
rename from api_mobile/response_data_types/references/__init__.py
rename to back_mobile/response_data_types/references/__init__.py
diff --git a/api_mobile/requests/__init__.py b/back_mobile/requests/__init__.py
rename from api_mobile/requests/__init__.py
rename to back_mobile/requests/__init__.py
diff --git a/api_mobile/response_data_types/registration/offer.py b/back_mobile/response_data_types/registration/offer.py
rename from api_mobile/response_data_types/registration/offer.py
rename to back_mobile/response_data_types/registration/offer.py
diff --git a/api_mobile/requests/auth/__init__.py b/back_mobile/requests/auth/__init__.py
rename from api_mobile/requests/auth/__init__.py
rename to back_mobile/requests/auth/__init__.py
diff --git a/api_mobile/response_data_types/registration/__init__.py b/back_mobile/response_data_types/registration/__init__.py
rename from api_mobile/response_data_types/registration/__init__.py
rename to back_mobile/response_data_types/registration/__init__.py
diff --git a/api_mobile/requests/payment/__init__.py b/back_mobile/requests/payment/__init__.py
rename from api_mobile/requests/payment/__init__.py
rename to back_mobile/requests/payment/__init__.py
diff --git a/api_mobile/requests/settings/__init__.py b/back_mobile/requests/settings/__init__.py
rename from api_mobile/requests/settings/__init__.py
rename to back_mobile/requests/settings/__init__.py
diff --git a/api_mobile/requests/main_page/__init__.py b/back_mobile/requests/main_page/__init__.py
rename from api_mobile/requests/main_page/__init__.py
rename to back_mobile/requests/main_page/__init__.py
diff --git a/api_mobile/response_data_types/product/deposits_result.py b/back_mobile/response_data_types/product/deposits_result.py
rename from api_mobile/response_data_types/product/deposits_result.py
rename to back_mobile/response_data_types/product/deposits_result.py
diff --git a/api_mobile/requests/references/__init__.py b/back_mobile/requests/references/__init__.py
rename from api_mobile/requests/references/__init__.py
rename to back_mobile/requests/references/__init__.py
diff --git a/api_mobile/requests/product/__init__.py b/back_mobile/requests/product/__init__.py
rename from api_mobile/requests/product/__init__.py
rename to back_mobile/requests/product/__init__.py
diff --git a/api_mobile/response_data_types/product/__init__.py b/back_mobile/response_data_types/product/__init__.py
rename from api_mobile/response_data_types/product/__init__.py
rename to back_mobile/response_data_types/product/__init__.py
diff --git a/api_mobile/requests/registration/__init__.py b/back_mobile/requests/registration/__init__.py
rename from api_mobile/requests/registration/__init__.py
rename to back_mobile/requests/registration/__init__.py
diff --git a/api_mobile/response_data_types/product/loans_result.py b/back_mobile/response_data_types/product/loans_result.py
rename from api_mobile/response_data_types/product/loans_result.py
rename to back_mobile/response_data_types/product/loans_result.py
diff --git a/api_mobile/test_data/__init__.py b/back_mobile/test_data/__init__.py
rename from api_mobile/test_data/__init__.py
rename to back_mobile/test_data/__init__.py
diff --git a/api_mobile/test_data/data/devices.json b/back_mobile/test_data/data/devices.json
rename from api_mobile/test_data/data/devices.json
rename to back_mobile/test_data/data/devices.json
diff --git a/api_mobile/test_data/data/users.py b/back_mobile/test_data/data/users.py
rename from api_mobile/test_data/data/users.py
rename to back_mobile/test_data/data/users.py
diff --git a/api_mobile/test_data/data/hosts.json b/back_mobile/test_data/data/hosts.json
rename from api_mobile/test_data/data/hosts.json
rename to back_mobile/test_data/data/hosts.json
diff --git a/api_mobile/test_data/data/__init__.py b/back_mobile/test_data/data/__init__.py
rename from api_mobile/test_data/data/__init__.py
rename to back_mobile/test_data/data/__init__.py
diff --git a/api_mobile/test_data/data/dfg.json b/back_mobile/test_data/data/dfg.json
rename from api_mobile/test_data/data/dfg.json
rename to back_mobile/test_data/data/dfg.json
diff --git a/api_mobile/test_data/data/paths.json b/back_mobile/test_data/data/paths.json
rename from api_mobile/test_data/data/paths.json
rename to back_mobile/test_data/data/paths.json
diff --git a/api_mobile/__init__.py b/back_mobile/__init__.py
rename from api_mobile/__init__.py
rename to back_mobile/__init__.py
diff --git a/api_mobile/test_data/db_models/__init__.py b/back_mobile/test_data/db_models/__init__.py
rename from api_mobile/test_data/db_models/__init__.py
rename to back_mobile/test_data/db_models/__init__.py
diff --git a/api_mobile/tests/__init__.py b/back_mobile/tests/__init__.py
rename from api_mobile/tests/__init__.py
rename to back_mobile/tests/__init__.py
diff --git a/api_mobile/response_data_types/__init__.py b/back_mobile/response_data_types/__init__.py
rename from api_mobile/response_data_types/__init__.py
rename to back_mobile/response_data_types/__init__.py
diff --git a/api_mobile/tests/steps/__init__.py b/back_mobile/tests/steps/__init__.py
rename from api_mobile/tests/steps/__init__.py
rename to back_mobile/tests/steps/__init__.py
